packages/manager/modules/vps/src/dashboard/migrate/migrate.service.js

```diff
@@ -18,7 +18,7 @@ export default class {
       .then(({ data }) => data);
   }
 
-  fetchNewPrice(newPlan, ovhSubsidiary, vpsMigration) {
+  fetchNewPrice(ovhSubsidiary, vpsMigration, datacenter) {
```

Comment:
**nitpick:** `vpsMigration` variable name does not make any sense since it actually holds the plan information.

Reply:
Renames it as "planInformation" as suggested.